runway/hooks/staticsite/auth_at_edge/templates/check_auth/__init__.py

```diff
@@ -0,0 +1,252 @@
+
+# pylint: skip-file
+"""Check the authorization information passed along via the cookie headers.
+
+When the information is not present or an error occurs due to verification a new
+request to Cognito will be made to authorize the user. The user will be taken
+to the Cognito login page to enter their information, and so long as it is valid,
+the information will be passed to the parsing agent.
+
+If a refresh token exists and it has expired (after 1 hour) do an automatic refresh
+of the credentials by redirecting the user to the refresh agent.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+import base64
+import datetime
+import hashlib
+import json
+import logging
+import random
+import re
+from urllib.parse import quote_plus, urlencode
+
+from jose import jwt  # pylint: disable=import-error
+
+from jwks_rsa.client import JwksClient  # pylint: disable=import-error
+from shared import decode_token, extract_and_parse_cookies, get_config  # noqa pylint: disable=import-error
+
+LOGGER = logging.getLogger(__file__)
+
+SECRET_ALLOWED_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~"
+NONCE_LENGTH = 16
+PKCE_LENGTH = 43
+
+
+def handler(event, _context):
+    """Handle the request passed in.
+
+    Keyword Args:
+        event (Dict[str, Any]): The Lambda Event
+    """
+    request = event['Records'][0]['cf']['request']
+    config = get_config()
```

Comment:
Since get_config only returns static values, can we move it outside the handler, e.g. 

`CONFIG = get_config()`

Probably wouldn't even be able to measure the performance difference if you tried, but it's the right principle.

Reply:
Good catch, that's something I forgot to update after the fact.